seqr/views/utils/pedigree_image_utils_tests.py

```diff
@@ -1,74 +1,83 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import unicode_literals
+
 from django.test import TestCase
 
+import mock
+import os
+
 from seqr.models import Family, Individual, Sample
-from seqr.views.utils.pedigree_image_utils import _get_parsed_individuals
+from seqr.views.utils.pedigree_image_utils import update_pedigree_images
 
 
 class PedigreeImageTest(TestCase):
     fixtures = ['users', '1kg_project']
 
-    def test_get_parsed_individuals(self):
+    @mock.patch('seqr.views.utils.pedigree_image_utils.BASE_DIR', '/')
+    @mock.patch('seqr.views.utils.pedigree_image_utils.random.randint')
+    @mock.patch('seqr.views.utils.pedigree_image_utils.os.system')
+    @mock.patch('seqr.views.utils.pedigree_image_utils.tempfile')
+    def test_update_pedigree_images(self, mock_tempfile, mock_os_system, mock_randint):
+        mock_tempfile.gettempdir.return_value = '/tmp'
```

Comment:
Codacy found an issue: [Probable insecure usage of temp file/directory.](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5795824)

Reply:
not relevant